src/nunit.gui.core/Views/IMainView.cs

```diff
@@ -36,8 +47,8 @@ public interface IMainView : IView
         FormWindowState WindowState { get; set; }
         Font Font { get; set; }
 
-        event FormClosingEventHandler FormClosing;
-        event DragEventHandler DragDrop;
+        event MainViewClosingEvent MainViewClosing;
+        event FilesDragAndDroppedEvent FilesDragAndDropped;
```

Comment:
The handler is a handler, not an event. 😄 I would keep the same names as windows, but in our namespace.